packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: External Identity Providers
+---
+# External Identity Providers
+As a developer building a custom application, you want to give your users the freedom to choose which Identity Provider that they use to sign in to your application. But first you should understand how various Identity Providers connect to Okta. 
+
+What is an Identity Provider? It is a service that creates and maintains identity information and then provides authentication services to your applications. Identity Providers can significantly reduce sign-in and registration friction, which allows your users to access applications with one click and without needing to create new passwords or remember usernames. 
```

Comment:
is it truly "one click" or perhaps 2? one to get to the IDP login modal, and one to submit?

Reply:
I'll change it to:

... Identity Providers can significantly reduce sign-in and registration friction, which allows your users to easily access applications without needing to create new passwords or remember usernames.

Reply:
sounds good, just being cautious in case people call out a technicality.